drivers/dali/Kconfig

```
@@ -0,0 +1,27 @@
+# DALI configuration options
+
+# Copyright (c) 2025 by Sven Hädrich <sven.haedrich@sevenlab.de>
+# SPDX-License-Identifier: Apache-2.0
+
+config DALI
+	bool "Digital Addressable Lighting Interface (DALI) drivers"
+	default y
+	help
+	  Enable generic support for DALI bus (IEC 62386).
+
+if DALI
+
+config MAX_FRAMES_IN_QUEUE
+	int "Max frames in DALI recv queue"
+	default 3
+	range 1 16
+	help
+	  How many frames should be buffered in the recv queue
```

Prefix DALI symbols with `DALI_`

```suggestion
config DALI_MAX_FRAMES_IN_QUEUE
	int "Max frames in DALI receive queue"
	default 3
	range 1 16
	help
	  How many frames should be buffered in the receive queue
```

---

Done that.